src/Servers/IIS/AspNetCoreModuleV2/OutOfProcessRequestHandler/outprocessapplication.cpp

```diff
@@ -87,7 +87,7 @@ OUT_OF_PROCESS_APPLICATION::SetWebsocketStatus(
     PCWSTR pszTempWebsocketValue;
     DWORD cbLength;
 
-    if (FAILED_LOG(pHttpContext->GetServerVariable("WEBSOCKET_VERSION", &pszTempWebsocketValue, &cbLength)))
+    if (pHttpContext->GetServerVariable("WEBSOCKET_VERSION", &pszTempWebsocketValue, &cbLength))
```

Comment:
This at least needs FAILED() to work correctly. But it's better to handle error code for not found result and  keep FAILED_LOG in other cases.

Reply:
It's `ERROR_INVALID_INDEX`